accurics-aws-tf-sample.tf

```diff
@@ -126,9 +126,21 @@ resource "aws_s3_bucket" "accurics-test-s3bucket1" {
   }
 
   tags = {
-    Name = format("%s-s3bucket1", var.acqaPrefix)
+    Name         = format("%s-s3bucket1", var.acqaPrefix)
     ACQAResource = "true"
-    Owner = "Accurics"
+    Owner        = "Accurics"
+  }
```

Comment:
**Accurics remediation**
 
**[HIGH]: S3 bucket should have versioning enabled**
 
**Vulnerability**
Enabling S3 versioning will enable easy recovery from both unintended user actions, like deletes and overwrites
 
**Remediation**
S3 bucket should have versioning enabled

source: accurics-app